exec/java-exec/src/main/java/org/apache/drill/exec/record/RecordBatchSizer.java

```diff
@@ -36,13 +39,17 @@
 import org.apache.drill.exec.vector.VariableWidthVector;
 
 import com.google.common.collect.Sets;
+import org.apache.drill.exec.vector.complex.RepeatedVariableWidthVectorLike;
 
 /**
  * Given a record batch or vector container, determines the actual memory
  * consumed by each column, the average row, and the entire record batch.
  */
 
 public class RecordBatchSizer {
+  private static final int OFFSET_VECTOR_WIDTH = 4;
```

Comment:
Perhaps use `UInt4Vector.VALUE_WIDTH` to initialize this constant. Use `UInt1Vector.VALUE_WIDTH` for the bit vector width.